### DISCUSSION TITLE
WebDAV integration

### DISCUSSION THREAD

**Discussion by Fishy78** [keepassxreboot/keepassxc#7421] [Ideas]
ATM it is only possible to connect to a Database, which is reachable via the local filesystem. Doesn't it make sense to increase the availability by using e.g. a secured NextCloud account natively? Today I am connecting the WebDAV via "net use", but that requires a permanent share, which needs to be set up every start (although it is marked as permanent). That is not very comfortable.

TL:NR: Is it possible to integrate a native WebDAV client like in keepass?

**Comment by droidmonkey** [comment]
See https://github.com/keepassxreboot/keepassxc/pull/7222 for movement in this space
